test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -7,19 +7,20 @@
 using System.IO;
 using System.Linq;
 using System.Threading;
-using NuGet.XPlat.FuncTest;
-using NuGet.Test.Utility;
-using NuGet.Packaging;
+
+using NuGet.Common;
 using NuGet.Packaging.Core;
-using NuGet.Packaging.PackageExtraction;
-using NuGet.Protocol;
+using NuGet.Test.Utility;
+using NuGet.XPlat.FuncTest;
+
```

Comment:
Did VS add the whitespace between the using statements, or did you? What's the pattern? `System.*` usings, whitespace, `NuGet.*` usings, whitespace, all other usings? Or usings grouped by first letter of namespace, each group alphabetical, except `System.*` first?